configs/experiment/local/eval_scale1_new.yaml

```diff
@@ -28,17 +28,17 @@ trainer:
 
 model:
   compile: True
-  #save_dir: /allen/aics/assay-dev/users/Suraj/EMT_Work/image_analysis_test/EMT_image_analysis/Colony_mask_training_inference/eval_whole_movie_multiscale_patch1_zarr_aws
+  save_dir: /allen/aics/assay-dev/users/Suraj/EMT_Work/image_analysis_test/EMT_image_analysis/Colony_mask_training_inference/eval_whole_movie_multiscale_patch1_zarr_aws
```

Comment:
If any configs use an `/allen` path, then the instructions need to tell people to edit it. Instead, I recommend using relative paths (starting with `./`) for portability. I've [filed an issue](https://github.com/AllenCell/EMT_image_analysis/issues/24) to track this going forward.